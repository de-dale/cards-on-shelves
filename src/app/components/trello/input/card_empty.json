{
  "id": "5be546175d23f22b8f7bd530",
  "desc": "",
  "descData": {
    "emoji": {}
  },
  "due": null,
  "email": null,
  "idBoard": "59379a9f9e21145b1904c8b1",
  "idChecklists": [],
  "idList": "5bf332375d40bf05681cac47",
  "idMembers": [],
  "idMembersVoted": [],
  "idShort": 64,
  "idAttachmentCover": null,
  "labels": [

  ],
  "idLabels": [
    "59379a9fced82109fff1ead6",
    "5bed2e3421173425ccbb7a6c"
  ],
  "manualCoverAttachment": false,
  "name": "Par d√©faut",
  "pos": 21888,
  "shortLink": "Jk3jnogk",
  "shortUrl": "https://trello.com/c/Jk3jnogk",
  "subscribed": false,
  "url": "https://trello.com/c/Jk3jnogk/64-par-d%C3%A9faut"
}