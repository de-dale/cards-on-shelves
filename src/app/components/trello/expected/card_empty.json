{
  "entities": [
    {
      "children": [
        1
      ],
      "id": 2,
      "name": "Par défaut",
      "type": "card"
    },
    {
      "content": "Par défaut",
      "id": 1,
      "type": "title"
    }
  ]
}